tyr/servers/server.py

```diff
@@ -701,6 +701,9 @@ def bake(self):
 
             self.chef_node.run_list = self.CHEF_RUNLIST
 
+            if self.chef_node.chef_environment == 'prod':
+                self.chef_node.run_list.append('role[RoleSumoLogic]')
```

Comment:
I think you won't need to do this.  In role_base::default, role_base::sumologic is included for all production nodes.  For Redhat nodes in EC2 this will add /var/log/messages and /var/log/secure to be indexed.  Per role_sumologic's readme, you can specify additional sources via data bag, but you won't need to include this to get the base stuff for a production node.